### PR TITLE
Add enpoint for reports in main api domain

### PR DESCRIPTION
**Pull request by kjgarza**
child of datacite/sashimi#30

### CODE REVIEW

File: prod-eu-west/services/metrics-api/main.tf [hunk 1]
```diff
@@ -35,7 +35,12 @@ resource "aws_lb_listener_rule" "metrics-api" {
 
   condition {
     field  = "host-header"
-    values = ["${aws_route53_record.metrics-api.name}"]
+    values = ["api.datacite.org"]
```
Comment (mfenner) [inline prod-eu-west/services/metrics-api/main.tf]: avoid hard-coding values and use `data.aws_route53_record.api.name`.

File: stage/services/metrics-api/main.tf [hunk 2]
```diff
@@ -35,7 +35,12 @@ resource "aws_lb_listener_rule" "metrics-api-stage" {
 
   condition {
     field  = "host-header"
-    values = ["${aws_route53_record.metrics-api-stage.name}"]
+    values = ["api.test.datacite.org"]
```
Comment (mfenner) [inline stage/services/metrics-api/main.tf]: See comment above.

File: prod-eu-west/services/metrics-api/main.tf [hunk 3]
```diff
@@ -35,7 +35,12 @@ resource "aws_lb_listener_rule" "metrics-api" {
 
   condition {
```
Comment (mfenner) [inline prod-eu-west/services/metrics-api/main.tf]: Make sure the priority is not already taken and is before any catch-all rule for api.datacite.org.

**Comment by mfenner** [comment]
@kjgarza please go ahead with this pull request.

**Comment by kjgarza** [comment]
parent datacite/sashimi#40

**Pull request merged**